pkg/sqlcache/schema_test.go

```diff
@@ -0,0 +1,231 @@
+package sql
```

Comment:
Same as above - I'd like to see this in `pkg/sqlcache/schematracke`

Reply:
Good point done.